### PR TITLE
Quan q5

### PR DESCRIPTION
**Pull request by Hedgemon4**

### CODE REVIEW

File: src/q5/Question5.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package q5;
+
```
Comment (Hedgemon4) [inline src/q5/Question5.java:2]: Include question text

File: src/q4/Question4.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package q4;
+
```
Comment (Hedgemon4) [inline src/q4/Question4.java:2]: Not sure if they wanted each question to branch of main or build on each other

File: src/q5/Question5Test.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+        int[] armStrongNums = {1, 2, 3, 4, 5, 6, 7, 8, 9, 153, 370, 371, 407, 1634, 8208, 9474, 54748};
+        
+        for(int i = 0; i < armStrongNums.length; i++ ){
+            String expectedOut = armStrongNums[i] + " is an Armstrong Number!" + System.lineSeparator();
+            provideInput(armStrongNums[i] + System.lineSeparator());
+            Question5.main( new String[0] );
+            String actualOutput = getOutput();
+            actualOutput = actualOutput.split(System.lineSeparator())[1 + i * 2] + System.lineSeparator();
+            assertEquals(expectedOut, actualOutput, "The output is different");
```
Comment (Hedgemon4) [inline src/q5/Question5Test.java:50]: Need a more helpful test message to fix code

File: src/q5/Question5Test.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    public void OutputCheckNotArmStrongNums() {
+
+        int[] notArmStrongNums = {10, 234, 534, 2452};
+        for(int i = 0; i < notArmStrongNums.length; i++ ){
+            String expectedOut = notArmStrongNums[i] + " is NOT an Armstrong Number!" + System.lineSeparator();
+            provideInput(notArmStrongNums[i] + System.lineSeparator());
+            Question5.main( new String[0] );
+            String actualOutput = getOutput();
+            actualOutput = actualOutput.split(System.lineSeparator())[1 + i * 2] + System.lineSeparator();
+            assertEquals(expectedOut, actualOutput, "The output is different");
```
Comment (Hedgemon4) [inline src/q5/Question5Test.java:65]: Need a more helpful test message to fix code

File: src/q5/Question5Test.java [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        }
+
+    }
+
+    @Test
+    public void OutputCheckNotArmStrongNums() {
+
+        int[] notArmStrongNums = {10, 234, 534, 2452};
```
Comment (Hedgemon4) [inline src/q5/Question5Test.java:58]: Could test more nonArmStrongNums

File: src/q5/Question5Test.java (hunk 3 above)
Comment (Hedgemon4) [reply src/q5/Question5Test.java:50]: To fix this, I just randomly generated test input, so this way I could tell them the actual and expected output without them being able to hardcode what I was testing. This way they have a more concrete idea about what is going wrong with their code.

File: src/q5/Question5.java (hunk 1 above)
Comment (SanaBob) [reply src/q5/Question5.java:2]: good catch. I'll add that

File: src/q4/Question4.java (hunk 2 above)
Comment (SanaBob) [reply src/q4/Question4.java:2]: We have not really talked about file structure. I would specify it in the meeting tomorrow

Comment (SanaBob) [reply src/q4/Question4.java:2]: Oh I was wrong nvm. I will change that

File: src/q5/Question5Test.java (hunk 3 above)
Comment (SanaBob) [reply src/q5/Question5Test.java:50]: As this is armstrong numbers the test cases are hardcoded. I will just add the expected and actual output in the assert statement.

File: src/q5/Question5Test.java (hunk 5 above)
Comment (SanaBob) [reply src/q5/Question5Test.java:58]: I will add some more rn

File: src/q5/Question5Test.java (hunk 4 above)
Comment (SanaBob) [reply src/q5/Question5Test.java:65]: Same as above, I will just add the expected input/output

File: src/q5/Question5Test.java [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+        int[] notArmStrongNums = {10, 123, 152, 234, 372, 408, 500, 534, 690, 1425, 2323, 2452, 54753};
+        for(int i = 0; i < notArmStrongNums.length; i++ ){
+            String expectedOut = notArmStrongNums[i] + " is NOT an Armstrong Number!" + System.lineSeparator();
+            provideInput(notArmStrongNums[i] + System.lineSeparator());
+            Question5.main( new String[0] );
+            String actualOutput = getOutput();
+            actualOutput = actualOutput.split(System.lineSeparator())[1 + i * 2] + System.lineSeparator();
+            assertEquals(expectedOut, actualOutput, "Expected:" + expectedOut + "Actual:" + actualOutput);
+        }
+
```
Comment (SamamthaH) [inline src/q5/Question5Test.java:67]: Maybe add a test to check negative numbers?
